src/api/v0/limit.rs

```diff
@@ -5,21 +5,78 @@ pub type Metadata = crate::resources::Metadata;
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub enum Period {
+    #[serde(rename = "minute")]
```

Comment:
I think you can use `rename_all` applied to the enum above like in https://serde.rs/container-attrs.html